react/features/mobile/external-api/middleware.js

```diff
@@ -83,7 +83,7 @@ function _toErrorString(
         error
             ? typeof error === 'string'
                 ? error
-                : Error.prototype.toString(error)
+                : Error.prototype.toString.apply(error)
```

Comment:
@paweldomas and/or @saghul, please take this commit out of this PR and push it into master!